scripts/format.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+	yapf -i *.py
+
+	# format unittest files
+	cd ./tests
+	yapf -i -r *.py
+
+# check if source code is formatted.
+elif [ $1 == "check" ]
+then
+	yapf -d -r *.py
+	set -eo pipefail
+
+	cd ./tests
+	yapf -d *.py
```

Comment:
This also isn't resolved.

Reply:
I think this is a bit better

 yapf -i ../securedrop/*.py
 yapf -i ../securedrop/tests/*.py

Reply:
Why not use `-r` option so you only have to call once?

Reply:
It weirdly didn't work when I set the recursive flag. I tested it by changing the formatting on unit test files and source files and only source files changed: this what I had "yapf -i -r ../securedrop/*.py" which didn't work.

Reply:
Remember, the script is called from project root. So the relative paths to recurse on are `securedrop` and `bin`.

Reply:
I changed the relative paths to the following:
yapf -i -r ./securedrop
yapf -i -r ./bin/securedrop 

Hopefully this does the trick.

Reply:
You don't need the preceding `./`, and you can just put both directories on the same line so only one `yapf` call is needed.

Reply:
Ok. I think that should be set now

Reply:
You want to recurse on the `bin` directory, not the file `securedrop` in the `bin` directory.

Reply:
done